### DISCUSSION TITLE
Angular Demo

### DISCUSSION THREAD

**Discussion by vaughanknight** [Green-Software-Foundation/carbon-aware-sdk#33] [General]
A discussion on building out a demo with Angular:
* What could it be?
* Who is the audience?
* What skills do we need?
* Who could contribute?

**Comment by sampart** [comment]
I guess this partly depends on the answers to those questions, but was there a reason Angular was chosen over React? Angular still has plenty of use, but React is quite a lot more popular. It's the number 1 web framework [according to Stack Overflow](https://insights.stackoverflow.com/survey/2021#technology) and wins on [Google Trends](https://trends.google.com/trends/explore?date=today%205-y&q=%2Fg%2F11c0vmgx5d,%2Fg%2F11c6w0ddw9,%2Fm%2F012l1vxv) too.

**Reply by vaughanknight** [reply]
The is already a react demo underway.  But someone had asked for an angular demo, or if they could contribute, as their company was building out a lot more angular.

**Reply by sampart** [reply]
Cool, thanks for clarifying.
